src/main/java/de/tum/in/www1/artemis/service/connectors/GitService.java

```diff
@@ -518,6 +530,8 @@ public Path getDefaultLocalPathOfRepo(VcsRepositoryUrl targetUrl) {
      * @return path of the local file system
      */
     public Path getLocalPathOfRepo(String targetPath, VcsRepositoryUrl targetUrl) {
+        if (targetUrl == null)
+            return null;
```

Comment:
```suggestion
        if (targetUrl == null) {
            return null;
        }
```
The code style checker probably wants braces here.